components/Dialog.vue

```diff
@@ -90,6 +91,12 @@ export default class Dialog extends Vue {
   // Content text.
   @Prop({ default: '' }) readonly text!: string
 
+  // Close the dialog by clicking outside
+  @Prop({ default: true }) readonly isClickToClose!: boolean
```

Comment:
Set to true to disable close by backdrop click
```suggestion
  @Prop({ default: true }) readonly isDisabledBackdropClick: boolean
```